andela/healthchecks_a
# #131049219 Ft job notification

**What does this PR do?**

Adds a new notification for a job that is running too often.

**Description of Task to be completed?**

It is supposed to add the feature of a job getting an icon status indicating that it is running too often.

**How should this be manually tested?**

Make a HTTP GET request to the localhost address of cronchecks.heroku.app, an excess number of times. (with reference to the stipulated Period and Grace period set. The icon for the respective check should turn to a blue icon.  

**Any background context you want to provide?**

Getting error: AssertionError: False is not true : Couldn't find 'icon-up' in response

**What are the relevant pivotal tracker stories?**
#131049219

**Screenshots** 

![no_icon_up_error_message](https://cloud.githubusercontent.com/assets/12303856/19264819/0a80a282-8fab-11e6-90e8-5798b37c6bdf.png)
Error message 

![no_icon_up](https://cloud.githubusercontent.com/assets/12303856/19264728/9a2c9734-8faa-11e6-890b-f7fed59df67c.png)
The Demo 2 check icon here should be a green icon (signifying that the status is up), instead it is blue.
